server/controllers/EmailController.js

```diff
@@ -0,0 +1,48 @@
+import sgMail from '@sendgrid/mail';
+import jwt from 'jsonwebtoken';
+
+require('dotenv').config();
+
+const key = process.env.SENDGRID_API_KEY;
+const cert = process.env.SECRET;
+const appUrl = process.env.APPURL;
+/**
+ *
+ *
+ * @class UserController
+ */
+class EmailController {
+/**
```

Comment:
#### Nitpick
Reformat the documentation
i.e.
```js
/**
       *
       *
       * @static
       * @param {object} user
       * @return {boolean} result
       * @description Sending email to users that signup
       * @memberof EmailController
       */
```

should be
```js
/**
* @static
* @param {object} user
* @return {boolean} result
* @description Sending email to users that signup
* @memberof EmailController
*/
```

Reply:
note